### DISCUSSION TITLE
Audio files open in fullscreen

### DISCUSSION THREAD

**Discussion by evilblade666** [smplayer-dev/smplayer#707] [Q&A]
Hi, folks!
Is there a way to make SMPlayer distinguish between audio and video? If I check the "Start videos in fullscreen" box - it also starts opening pictures of mp3 files (if they have one) in fullscreen.

**Comment by Raza-0007** [comment]
Is this setting not working?

Preference>Interface>Interface>Hide video window when playing audio files

**Reply by evilblade666** [reply]
> Is this setting not working?
> Preference>Interface>Interface>Hide video window when playing audio files

Nope. It was checked by default and I've also tried to uncheck and then check it back again.. No changes sadly :(

**Comment by smplayer-dev** [comment]
That's because if the audio has a picture attached, it is as if it had a video track, so for SMPlayer it's a video.

**Reply by evilblade666** [reply]
> That's because if the audio has a picture attached, it is as if it had a video track, so for SMPlayer it's a video.

I've got this already, thanks :) My question is - can I make SMPlayer somehow "understand" that it is an audio, not a video, and there's no need to make it fullscreen?

**Comment by smplayer-dev** [comment]
You can pass to mpv the option `--no-video` to disable the video track.
